package/network/config/qos-scripts/files/usr/lib/qos/generate.sh

```diff
@@ -326,7 +326,8 @@ start_interface() {
 			append cstr "$classnr:$prio:$avgrate:$pktsize:$pktdelay:$maxrate:$qdisc:$filter" "$N"
 		done
 		append ${prefix}q "$(tcrules)" "$N"
-		export dev_${dir}="ip link set $dev up >&- 2>&-
+		export dev_${dir}="ip link add ${dev} type ifb >&- 2>&-
+ip link set $dev up >&- 2>&-
```

Comment:
why no ident?

Reply:
Because I kept with the style which was already done before.

Reply:
Wow looking at this we should do some shell script linting...